### PR TITLE
Create Cache Dismiss Function

### PR DESCRIPTION
**Pull request by thisnugroho**
Related with issue #16

### CODE REVIEW

File: KeywordCloudBlockPlugin.inc.php [hunk 1]
```diff
@@ -39,6 +39,12 @@ function getDisplayName() {
 		return __('plugins.block.keywordCloud.displayName');
 	}
 
+	/**
+	 * fallBack for getFileCache
+	 */
+	function _cacheDismiss() {
```
Comment (ghost) [inline KeywordCloudBlockPlugin.inc.php:45]:
```suggestion
	function cacheDismiss() {
```
The use of an underscore prefix in method names has no meaning according to [PSR12 coding style](https://www.php-fig.org/psr/psr-12/)(take a look at section 4.4). 

Is there any particular reason that made you use it?

Comment (thisnugroho) [reply KeywordCloudBlockPlugin.inc.php:45]: It doesn't mean anythinf, thank you for your review 👍🙂

Comment (thisnugroho) [reply KeywordCloudBlockPlugin.inc.php:45]: > ```suggestion
> 	function cacheDismiss() {
> ```
> The use of an underscore prefix in method names has no meaning according to [PSR12 coding style](https://www.php-fig.org/psr/psr-12/)(take a look at section 4.4). 
> 
> Is there any particular reason that made you use it?  

It doesn't mean anything, thank you for your review 👍🙂

Comment (ghost) [reply KeywordCloudBlockPlugin.inc.php:45]: Can you send another commit, removing the underscore? So we can merge your contribution? 🙂

**Comment by ghost** [comment]
Well, thank you for this important contribution @thisnugroho! I'm going to merge the current code into our `main` branch

File: KeywordCloudBlockPlugin.inc.php (hunk 1 above)
Comment (thisnugroho) [reply KeywordCloudBlockPlugin.inc.php:45]: I've send another pull request for removing the underscore. #18

**Pull request merged**